packages/vue/src/cypress/FCalendarDay.pageobject.ts

```diff
@@ -13,7 +13,7 @@ export enum dayType {
 /**
  * @public
  */
-export class FCalenderDayPageobject implements BasePageObject {
+export class FCalendarDayPageobject implements BasePageObject {
```

Comment:
```suggestion
export class FCalendarDayPageObject implements BasePageObject {
```
Om vi ändå är och ändrar stavningen ska vi kanske ändra så att det är `PageObject` så som majoriteten av våra andra page objekt istället för `Pageobject`? Vad jag sett så är det `PageObject` som ska vara korrekt då det är särskrivet inom POM.

Reply:
Ändrade enligt ditt förslag till `FCalendarDayPageObject`.